### PR TITLE
edits to unconference page

### PR DESCRIPTION
**Pull request by rosewms**
Added a 'view only' version of the 2017 Unconference schedule.
Edited the language of the last sentence - now using the 2016 schedule as an example.

### CODE REVIEW

File: docs/conf/na/2017/unconference.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Schedule a Session
 ------------------
 
 Attendees that wish to lead an unconference session have several options
-for signing up. We will have a board with all of the sessions for the
+for signing up. We will have a 'board <https://docs.google.com/spreadsheets/d/1Im3Ce1-JubF33Ydg4XPCnafZbkVMKXDtWE9uoa_SskI/edit?usp=sharing>`__ with all of the sessions for the
```
Comment (thatdocslady) [inline docs/conf/na/2017/unconference.rst:49]: The link doesn't seem to resolve in rendering. The quotes need to be slanted (``) not straight ('') in the beginning and the end. Also, need to remove one underscode, but that's more for cleanness than function :)

File: docs/conf/na/2017/unconference.rst [hunk 2]
```diff
@@ -66,4 +66,4 @@ Unconference sessions will begin after lunch both days, running until
 the end of the day. We will start the unconference sessions off each day
 with some time for focus on one of several topics:
 
-Check out the `schedule <https://docs.google.com/spreadsheets/d/1LZ2MrNrQ4AWX6pwtrj08qr6dmNl9O0-kkuHxDNN3zzU/edit?usp=sharing>`__ or come down to Lola's room to propose a topic.
+Check out last years `schedule <https://docs.google.com/spreadsheets/d/1LZ2MrNrQ4AWX6pwtrj08qr6dmNl9O0-kkuHxDNN3zzU/edit?usp=sharing>`__.
```
Comment (thatdocslady) [inline docs/conf/na/2017/unconference.rst:69]: "last year's"

File: docs/conf/na/2017/unconference.rst (hunk 1 above)
Comment (thatdocslady) [inline docs/conf/na/2017/unconference.rst:49]: The link text is broken, need to replace the straight quote (') with a slanted quote (`) in the beginning, similar to the link at the bottom of the doc.

**Comment by plaindocs** [comment]
Hi @rosewms - do you want to fix these up according to @thatdocslady's changes, or shall I pick it up?

**Pull request merged**